### PR TITLE
Adding tests for Selling stock

### PR DESCRIPTION
**Pull request by Peaceful36**
Made some tests and added documentations for sell-stock use case.

### CODE REVIEW

**Review by inaam10-tech — CHANGES_REQUESTED** [review]
I believe you have one failed test, you might want to check on that before merging

**Comment by Peaceful36** [comment]
> I believe you have one failed test, you might want to check on that before merging

I have fixed the errors in my tests.

**Review by jackzxliu — APPROVED** [review]
Looks good

**Review by inaam10-tech — DISMISSED** [review]
Seems good now.

**Review by noel-toms — APPROVED** [review]
Everything seems good

**Review by eladperlman — CHANGES_REQUESTED** [review]
You should add more tests, also the numbers you use in your scenarios such as buying "4" stocks should be stored as a constant, I see those magic numbers all over the file.

File: src/test/java/SellStockTest/SellStockTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import entities.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.sql.Date;
+import java.util.Map;
+import java.util.Objects;
+
+public class SellStockTest {
```
[the remark this answers is not in the document]
Comment (noel-toms) [reply src/test/java/SellStockTest/SellStockTest.java:22]: To add to this I think it's very important to make tests concerning the database, as selling stocks removes the stock based on the quantity from the user. Something worth considering might be to make test tables in the database and fill them with values, and the assure that after selling a stock that the quantity of that stock has decreased.

**Comment by noel-toms** [comment]
> You should add more tests, also the numbers you use in your scenarios such as buying "4" stocks should be stored as a constant, I see those magic numbers all over the file.

To add to this I think it's very important to make tests concerning the database, as selling stocks removes the stock based on the quantity from the user. Something worth considering might be to make test tables in the database and fill them with values, and the assure that after selling a stock that the quantity of that stock has decreased.

**Review by inaam10-tech — DISMISSED** [review]
All good now

File: src/test/java/SellStockTest/SellStockTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Peaceful36) [reply src/test/java/SellStockTest/SellStockTest.java:22]: I have added test and made use of constants

**Review by Spaghetti-12 — COMMENTED** [review]
Looks good!

**Review by Spaghetti-12 — APPROVED** [review]

**Review by eladperlman — APPROVED** [review]
Looks good!

**Review by noel-toms — APPROVED** [review]
Approved.

**Pull request merged**